Examples/GithubPaginatedSearch.swift

```diff
@@ -134,17 +138,17 @@ class GithubPaginatedSearchViewController: UIViewController {
         }
 
         Driver.system(
-                initialState: State.empty,
-                reduce: State.reduce,
-                feedback:
-                    // UI, user feedback
-                    bindUI,
-                    // NoUI, automatic feedback
-                    react(query: { $0.loadNextPage }, effects: { resource in
-                        return URLSession.shared.loadRepositories(resource: resource)
-                            .asSignal(onErrorJustReturn: .failure(.offline))
-                            .map(Event.response)
-                    })
+            initialState: State.empty,
```

Comment:
Can you keep the indentation here the same as it was on the source? (see counterparts)

Reply:
Yeah definitely it was XCode indentation **Ctrl + I**